packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -22,273 +22,330 @@ related: [
-* A user leaving a required field blank.
-* A user leaving a required field incomplete.
+Errors can be validated at two points in the workflow: On loss of focus or on submission. The error state you use depends on whether validation happens on the client side (loss of focus) or server side (on submission).
 
-### Validation on submission
-Validation on submission is performed when the user attempts to submit a form. In this scenario, the user completes the form in its entirety and the page is reloaded with any errors detected. 
+### Error validation on loss of focus
+Validation on loss of focus occurs as soon as a field loses focus. It alerts users of errors immediately after they complete their input through inline validation, allowing them to quickly see and address issues.
 
-For validation on submission, use an [inline alert](/components/alert/design-guidelines#inline-alerts) along with field level errors. This combination of error types should make it simple for the user to quickly identify areas that need to be fixed. The inline alert should be placed below the form's header and description, and should take on the width of the form. Inline alerts should describe the general error, but refrain from going into great detail.
+Validate errors on loss of focus when:
 
-<img src="./img/inline_error.png" alt="Example of errors after validation on submission, including an inline alert at the top, and field level errors" width="551"/>
+* A user enters data in an invalid format.
+* A user leaves a required field blank.
+* A user leaves a required field incomplete.
 
-## Form formatting
-### Spacing
-There should always be 24px spacing underneath every form input. If there is helper text, the 24px spacing should start after the helper text. Additionally, the spacing between data inputs like checkboxes and radio buttons, for example, should also be 24px when on the same line and 24px when the options are presented on top of one another. For more spacing information, read the [PatternFly spacer guidelines](/guidelines/spacers#considering-line-height-and-padding).
+### Error validation on submission
+Validation on submission occurs when a user attempts to submit a form. In this scenario, the user completes the form in its entirety and the page reloads with any errors detected.
 
-<img src="./img/form-spacing.png" alt="Basic form example with spacers to demonstrate how form elements should be spaced" width="460"/>
+For validation on submission, supplement field level errors with anor validation on submission, use an [inline alert](/components/alert/design-guidelines#inline-alerts) to make it simple for the user to quickly identify and address areas that need attention. The inline alert should be placed below the form’s header and description, and it should equal the total width of the form. 
 
+<img src="./img/validation-submission.png" alt="Example of errors after validation on submission, including an inline alert at the top, and field level errors"/>
```

Comment:
Could you add a width string to the image string width="794"? The image looks huge right now haha

<img src="./img/validation-submission.png" alt="Example of errors after validation on submission, including an inline alert at the top, and field level errors" width="794" />